src/index.html

```diff
@@ -9,6 +9,129 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Human rights</h1>
+  <h1>Universal Declaration of Human Rights</h1>
+  <h2>Preamble</h2>
+    <p>Whereas recognition of the inherent dignity and of the equal and inalienable
```

Comment:
`p` is not a nested tag for `h2` so you don't need an extra indentation for it;
when you write some text inside tags move it to the new line and add proper indentations so that the code looks clear